lib/rendition.js

```diff
@@ -279,6 +282,26 @@ class Rendition {
         return obj;
     }
 
+    embed() {
```

Comment:
Maybe naming it `shouldEmbed()` or `shouldEmbedInIOEvent()` might be more self-explaining.

Reply:
+1 for `shouldEmbedInIOEvent()`